### DISCUSSION TITLE
gomod2nix maintainership

### DISCUSSION THREAD

**Discussion by brianmcgee** [nix-community/infra#1471] [General]
Hey,

I see PRs building up for `gomod2nix`, some trivial dependency changes, some stale, some already approved, and others looking to improve functionality. It seems that a few different people have been merging things ad hoc. The project was initially created and maintained by @adisbladis, but there's no longer an official maintainer. 

Based on recent merges, are any of you interested in taking on the maintainer role @marcusramberg, @jtcarnes, @ajaxbits? I know there's no point in asking @Mic92; he's busy maintaining half of the Open Source ecosystem by himself :wink: 

Is anyone else interested? 

I'm willing to consider taking it on myself, but I've only been a user up to now, and I see plenty of others who have been contributing that may be better placed if they can take it on.

**Comment by marcusramberg** [comment]
> Hey,
> 
> 
> 
> I see PRs building up for `gomod2nix`, some trivial dependency changes, some stale, some already approved, and others looking to improve functionality. It seems that a few different people have been merging things ad hoc. The project was initially created and maintained by @adisbladis, but there's no longer an official maintainer. 
> 
> 
> 
> Based on recent merges, are any of you interested in taking on the maintainer role @marcusramberg, @jtcarnes, @ajaxbits? I know there's no point in asking @Mic92; he's busy maintaining half of the Open Source ecosystem by himself :wink: 
> 
> 
> 
> Is anyone else interested? 
> 
> 
> 
> I'm willing to consider taking it on myself, but I've only been a user up to now, and I see plenty of others who have been contributing that may be better placed if they can take it on. 

Hey. I've now been given privs for this repo, and plan to merge some more PRs, currently in Portugal for a wedding, but soon I'll be back in front of a computer :)

**Reply by marcusramberg** [reply]
Would certainly appreciate any other contributors wanting to help out.

**Reply by brianmcgee** [reply]
I'm happy to help out, I use it in a fair number of projects now.

**Reply by Mic92** [reply]
I created now a team for it: https://github.com/orgs/nix-community/teams/gomod2nix/members
and invited @marcusramberg as well.

**Comment by ajaxbits** [comment]
I can help with typos and basic maintenance. But I'm not a heavy user of `gomod2nix`, so won't be able to contribute to more extensive functionality at this time.

**Comment by tscolari** [comment]
I would be happy to help too.
I'm experienced with Go, learning Nix as I go along. But I definitely can spend some time helping with PRs and maybe some bug issues.

**Comment by FKouhai** [comment]
I'd love to help out with that as well

**Comment by marcusramberg** [comment]
Apparently I can't add people to the team, but I'd be happy to have more comaintainers, @Mic92 can you help?

**Reply by adisbladis** [reply]
@marcusramberg  I've changed your permission level to admin on the repo and sent you an invite to the nix-community org.
